src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReferenceBases.java

```diff
@@ -22,14 +23,19 @@
 
     public static final int NUM_BASES_ON_EITHER_SIDE = 10;
 
+    protected final OneShotLogger warning = new OneShotLogger(this.getClass());
+
     @Override
     public List<String> getKeyNames() { return Collections.singletonList(REFERENCE_BASES_KEY); }
 
     @Override
     public Map<String, Object> annotate(final ReferenceContext ref,
                                         final VariantContext vc,
                                         final ReadLikelihoods<Allele> likelihoods) {
-
+        if (ref==null)  {
```

Comment:
The warning seems good for now, and consistent with existing behavior in the annotation classes -- we can open a separate ticket to discuss whether there should be a mode in which GATK throws in these cases.